Loader/Loader.cs

```diff
@@ -42,12 +44,14 @@ public static NativeCallbacks Load()
             //PluginsContainer.LoadPluginsFromSubDirs(myLocation);
 
             // Return the marshalled callbacks for the native interfaces
-            return new NativeCallbacks
+            i_callbacks = new NativeCallbacks
             {
                 NativeAPICallback = Marshal.GetFunctionPointerForDelegate<NativeAPICallback>(NativeAPIHandler),

```

Comment:
Change to:
NativeAPICallback = Marshal.GetFunctionPointerForDelegate<NativeAPICallback>(APICallback);
etc for the 3